core/components/archivist/elements/snippets/snippet.archivist.php

```diff
@@ -139,7 +139,7 @@
 $output = array();
 $groupByYearOutput = array();
 $idx = 0;
-$count = count($resources);
+$count = count(array($resources));
```

Comment:
Isn't that just gonna make count equal to 1 all the time, because it then contains an array with a single iterator object?

Reply:
Oops, yeah a better 'quick' fix would be switching getIterator to getCollection I suppose